### PR TITLE
[Merge] unhide `--Xvalidators-proposer-*`

### PR DESCRIPTION
**Pull request by tbenr**
## PR Description

unhide `--Xvalidators-proposer-*`

## Fixed Issue(s)
#4874

## Documentation

- [ ] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [ ] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorProposerOptions.java [hunk 1]
```diff
@@ -28,15 +28,15 @@
   private String proposerDefaultFeeRecipient = null;
 
   @Option(
-      names = {"--Xvalidators-proposer-config"},
+      names = {"--validators-proposer-config"},
       paramLabel = "<STRING>",
       description = "remote URL or local file path to load proposer configuration from",
       arity = "0..1",
       hidden = true)
```
Comment (ajsutton) [inline teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorProposerOptions.java]: Need to remove the hidden = true as well.

Comment (ajsutton) [reply teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorProposerOptions.java]: Which apparently we forgot for the ExecutionEngineOptions - could you unhide them too please?

**Review by ajsutton — APPROVED** [review]
One more to unhide please:
```
 @Option(
      hidden = true,
      names = {"--ee-jwt-secret-file"},
```

but LGTM otherwise. :)

**Comment by tbenr** [comment]
> One more to unhide please:

oh gosh is on the top 😭

**Pull request merged**